src/main/controllers/helpers/TribunalOrderOrRequestHelper.ts

```diff
@@ -307,3 +310,194 @@ function hasClaimantResponded(notification: SendNotificationType) {
+    },
+    {
+      key: {
+        text: translations.description,
+        classes: 'govuk-!-font-weight-regular-m',
+      },
+      value: {
+        text: response.value.respondNotificationUploadDocument?.find(element => element !== undefined).value
+          .shortDescription,
+      },
+    },
+    {
+      key: {
+        text: translations.document,
+        classes: 'govuk-!-font-weight-regular-m',
+      },
+      value: {
+        html: await getSupportingMaterialDownloadLink(
+          response.value.respondNotificationUploadDocument?.find(element => element !== undefined).value
+            .uploadedDocument,
+          accessToken
+        ),
+      },
+    },
+    {
+      key: {
+        text: translations.requestMadeBy,
+        classes: 'govuk-!-font-weight-regular-m',
+      },
+      value: {
+        text:
+          response.value.respondNotificationRequestMadeBy === 'Request'
+            ? translations[response.value.respondNotificationRequestMadeBy]
+            : translations[response.value.respondNotificationCaseManagementMadeBy],
+      },
+    },
+    {
+      key: {
+        text: translations.fullName,
+        classes: 'govuk-!-font-weight-regular-m',
+      },
+      value: { text: response.value.respondNotificationFullName },
+    },
+    {
+      key: {
+        text: translations.sentTo,
+        classes: 'govuk-!-font-weight-regular-m',
+      },
+      value: { text: translations[response.value.respondNotificationPartyToNotify] },
+    },
+  ]);
+};
+
+async function addTribunalResponses(
+  adminResponseCollection: TypeItem<RespondNotificationType>[],
+  req: AppRequest<Partial<AnyRecord>>,
+  allResponses: any[],
+  translations: AnyRecord
+) {
+  for (const response of adminResponseCollection) {
+    const responseDate = datesStringToDateInLocale(response.value.respondNotificationDate, req.url);
+    const responseToAdd = await addAdminResponse(
+      allResponses,
+      translations,
+      response,
+      req.session.user?.accessToken,
+      responseDate
+    );
+    if (responseToAdd !== undefined) {
+      allResponses.push(responseToAdd);
+    }
+  }
+}
+
+async function addClaimantRespondentResponses(
+  respondCollection: TypeItem<PseResponseType>[],
+  req: AppRequest<Partial<AnyRecord>>,
+  translations: AnyRecord,
+  allResponses: any[]
+) {
+  for (const response of respondCollection) {
+    const responseDate = datesStringToDateInLocale(response.value.date, req.url);
+    let responseToAdd;
+    if (
+      response.value.from === Applicant.CLAIMANT ||
+      (response.value.from === Applicant.RESPONDENT && response.value.copyToOtherParty === YesOrNo.YES)
+    ) {
+      responseToAdd = await addNonAdminResponse(translations, response, req.session.user?.accessToken, responseDate);
+    }
+    if (responseToAdd !== undefined) {
+      allResponses.push(responseToAdd);
+    }
+  }
+}
+
+export const getClaimantTribunalResponseBannerContent = (
```

Comment:
Increase the test coverage here - https://sonarcloud.io/code?id=et-sya&pullRequest=1146&selected=et-sya%3Asrc%2Fmain%2Fcontrollers%2Fhelpers%2FTribunalOrderOrRequestHelper.ts&line=491

Have some tests that call this method directly

Reply:
Added coverage 
https://sonarcloud.io/component_measures?metric=new_coverage&selected=et-sya%3Asrc%2Fmain%2Fcontrollers%2Fhelpers%2FTribunalOrderOrRequestHelper.ts&view=list&pullRequest=1146&id=et-sya